### PR TITLE
Fixes #12910: Remove plugin-common from rudder-plugins

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/12910

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

File: makefiles/global-vars.mk [hunk 1]
```diff
@@ -5,7 +5,6 @@
 
 MAIN_BUILD          = $(shell  [ -f main-build.conf ] && echo '.' || echo '..')/main-build.conf
 RUDDER_BRANCH       = $(shell sed -ne '/^rudder-branch=/s/rudder-branch=//p' $(MAIN_BUILD))
-LIB_COMMON_VERSION  = $(shell sed -ne '/^lib-common=/s/lib-common=//p' $(MAIN_BUILD))
 LIB_PRIVATE_VERSION = $(shell sed -ne '/^lib-common-private=/s/lib-common-private=//p' $(MAIN_BUILD))
 PARENT_VERSION      = $(shell sed -ne '/^parent-plugin=/s/parent-plugin=//p' $(MAIN_BUILD))
```
Comment (fanf) [inline makefiles/global-vars.mk]: Can you add:  
BUILD_VERSION       = $(shell sed -ne '/^rudder-build-version=/s/rudder-build-version=//p' $(MAIN_BUILD)) 

here ?

File: makefiles/global-vars.mk [hunk 2]
```diff
@@ -27,7 +25,6 @@ generate-pom:
 	cp pom-template.xml pom.xml
 	sed -i -e "s/\$${rudder-branch}/$(RUDDER_BRANCH)/" pom.xml
 	sed -i -e "s/\$${parent-version}/$(PARENT_VERSION)/" pom.xml
-	sed -i -e "s/\$${lib-common}/$(LIB_COMMON_VERSION)/" pom.xml
 	sed -i -e "s/\$${lib-common-private}/$(LIB_PRIVATE_VERSION)/" pom.xml
 	sed -i -e "s/\$${plugin-version}/$(VERSION)/" pom.xml
```
Comment (fanf) [inline makefiles/global-vars.mk]: And here: 
	sed -i -e "s/\$${rudder-build-version}/$(BUILD_VERSION)/" pom.xml

File: pom-template.xml [hunk 3]
```diff
@@ -12,18 +12,14 @@
     <groupId>com.normation.rudder</groupId>
     <artifactId>rudder-parent</artifactId>
     <!-- here, we can't use ${rudder-build-version} because of maven limitation-->
-    <version>4.3.0</version>
+    <version>4.3.3-SNAPSHOT</version>
```
Comment (fanf) [inline pom-template.xml]: And use it here: 
    <version>${rudder-build-version}</version>

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**